### PR TITLE
Completed css-canvas-medical exercise. - Justin Son

### PR DESCRIPTION
**Pull request by justintaeson**
css-canvas-medical_B_2 2 22
css-canvas-medical_A_2 2 22

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div id="container">
+    <div class="row">
+      <div class="column">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:15]: `column` what? Did you only need one size of `column` for your entire layout? How wide is the `column` meant to be?

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        <p class="navbar-text">Implementation</p>
+        <p class="navbar-text">About Us</p>
+        <p class="navbar-text">Blog<p>
+        <button class="button">Contact Us</button>
+      </div>
+    </div>
+    <hr id="navbar-line">
+    <br>
+    <br>
+    <br>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:29]: No `br` elements, if you need additional space, you should use `margin` or `padding`

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+}
+
+.row {
+  display: flex;
+  margin: auto;
+}
+
+.column {
+  margin: auto;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:81]: `column` classes are meant to have either `flex-basis` or `width`, and that's it. This is not a proper layout class setup, I suggest reviewing `css-layout-classes` for a refresher.

File: css-canvas-medical/styles.css [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+  right: 200px;
+}
+
+#white-logo {
+  width: 140px;
+  height: 30px;
+  position: absolute;
+  top: 110px;
+  left: 80px;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:228]: None of these elements should require using `absolute` positioning, you should use proper layout classes.

File: css-canvas-medical/styles.css [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+.canvas-info {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 250;
+  color: white;
+  font-size: small;
+  position: absolute;
+  top: 200px;
+  left: 80px;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:239]: This shouldn't be `absolute` either 🤢

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**